src/test/frontend/db/model_test.cljs

```diff
@@ -0,0 +1,84 @@
+(ns frontend.db.model-test
+  (:require [frontend.db.model :as model]
+            [frontend.db.conn :as conn]
+            [datascript.core :as d]
+            [frontend.db-schema :as schema]
+            [frontend.handler.repo :as repo-handler]
+            [cljs.test :refer [deftest is are testing]]))
+
+(defonce test-db "test-db")
+
+(defn- run-db!
```

Comment:
Is there  a `use-fixture` feature in cljs?

Reply:
yes, there's one, we could use it here.